daemon/daemon_main.go

```diff
@@ -288,6 +288,9 @@ func init() {
 	flags.Bool(option.EnableHealthChecking, defaults.EnableHealthChecking, "Enable connectivity health checking")
 	option.BindEnv(option.EnableHealthChecking)
 
+	flags.StringSlice(option.EndpointStatus, []string{}, "Enable additional CiliumEndpoint status features")
```

Comment:
Should this include example values or at least a pointer to what those value can be?